lib/jxl/dec_frame.h

```diff
@@ -54,8 +54,11 @@ class FrameDecoder {
  public:
   // All parameters must outlive the FrameDecoder.
   FrameDecoder(PassesDecoderState* dec_state, const CodecMetadata& metadata,
-               ThreadPool* pool)
-      : dec_state_(dec_state), pool_(pool), frame_header_(&metadata) {}
+               ThreadPool* pool, bool use_slow_rendering_pipeline)
```

Comment:
It might be more convenient to not add this to the constructor but have a `SetUseRenderingPipeline` function that doesn't take a bool argument but an int/enum one (kDisabled, kSlow, kLowMemory) so later the fancy pipeline can also be tested without having to change the constructor again.

Reply:
I don't envision a state in which both pipelines will coexist with the current path - rather, once the low-memory pipeline is implemented and the slow pipeline can do both, we'll remove the current path and have just the two pipelines.